aw_watcher_window/main.py

```diff
@@ -27,6 +27,10 @@ def main():
     if sys.platform.startswith("linux") and ("DISPLAY" not in os.environ or not os.environ["DISPLAY"]):
         raise Exception("DISPLAY environment variable not set")
 
+    if sys.platform == "darwin":
+        from . import macos
```

Comment:
Maybe do this after logging has been set up? Just in case.

Reply:
Fair point. I moved it there to have it closer to the other system independent code in main.py but putting it after logging is a better idea

Reply:
Fixed in b829e04